src/commands/branches.test.ts

```diff
@@ -26,6 +26,47 @@ describe('branches', () => {
     },
   });
 
+  testCliCommand({
+    name: 'create branch and connect with psql',
+    args: [
+      'branches',
+      'create',
+      '--project-id',
+      'test',
+      '--name',
+      'test_branch',
+      '--psql',
+    ],
+    env: {
+      PATH: `mocks/bin:${process.env.PATH}`,
```

Comment:
I think it's ok to add this to `testCliCommand` code, it's pretty convenient way to mock external tools! And we most likely would never need to switch between mocked and real binary